uPortal-rendering/src/main/java/org/apereo/portal/url/UrlSyntaxProviderImpl.java

```diff
@@ -431,6 +431,7 @@ public IPortalRequestInfo getPortalRequestInfo(HttpServletRequest request) {
                                 break;
                             }
                         }
+                        break;
```

Comment:
It is unclear if these breaks were accidentally or intentionally missed. These may need to be removed and replaced by a `@SuppressWarnings("FallThrough")`.